_episodes/first_example_recipe.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Running a recipe (First example)"
+teaching: 20
+exercises: 25
+questions:
+- "What is a recipe?"
+- "How can I do the same preprocessing on many different datasets?"
+- "What are the files and directories after running a recipe?"
```

Comment:
If it does not make this question too long I would rephrase "What are the files and directories that are created after running a recipe?"

Reply:
changed in last commit